.github/workflows/docs.yml

```diff
@@ -25,8 +25,9 @@ jobs:
 
       - name: Install dependencies
         run: |
-          pip install pdocs kubernetes loguru pyyaml
-          python setup.py install
+          pip install pdocs kubernetes loguru pyyaml packaging hatch 
```

Comment:
It's cool decouple ci from dependencies and less error prone. 
So if you build with hatch you can install that build and avoid other pip installs